web_scraping.py

```
@@ -0,0 +1,31 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+def scrape_helicopter_escapes(output_file="helicopter_prison_escapes.csv"):
+    """
+    Scrapes the Wikipedia page of helicopter prison escapes and saves it as CSV.
+    """
+    url = "https://en.wikipedia.org/wiki/List_of_helicopter_prison_escapes"
+    response = requests.get(url)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)